payloads/src/payload/mod.rs

```diff
@@ -1,5 +1,5 @@
-mod payload;
+mod payload_private;
 // TODO: serialize doesn't even compile yet
 // mod serialize;
 
-pub use payload::*;
+pub use payload_private::*;
```

Comment:
Could you just move payload.rs into this file and delete payload.rs?

Reply:
+1